opam-ci-check/README.md

```diff
@@ -0,0 +1,31 @@
+# opam-ci-check
+
+## Synopsis
+
+A CLI tool that tests whether packages are publishable on the opam repository.
+
+This tool is in progress.
+
+## Installation
+
+``` sh
+opam install https://github.com/ocurrent/opam-ci-check.git
+```
+
+## Documentation
+
+https://www.ocurrent.org/opam-ci-check/
+
+## Contribution
+
+Contributions are most welcome!
+
+<!-- TODO -->
+- [File issues](https://github.com/ocurrent/opam-ci-check/issues) to report bugs or feature requests.
+- [Contribute code or documentation](./CONTRIBUTING.md)
+
+---
+
+This project is created and maintained by\
```

Comment:
Yeah, I thought about that! I only hesitated because I was not sure at the time whether that is the consistent with the long term aim of eventually getting the infrastructure to be a community supported effort.

Reply:
Makes sense! :)